frontend/src/components/CreateGoal/ObjectiveFiles.scss

```diff
@@ -0,0 +1,3 @@
+.ttahub-objective-files-tool-tip {
+    font-size: .8rem;
```

Comment:
This looks weird, them having different sizes

![Screen Shot 2022-07-20 at 1 23 22 PM](https://user-images.githubusercontent.com/3288586/180044331-a28b86fc-1e69-484e-95c1-6809d3ae10cc.png)